notebooks/266-speculative-sampling/266-speculative-sampling.ipynb

```diff
@@ -23,8 +23,6 @@
    "id": "f97c435a",
```

Comment:
<p>are you sure that cleanup should be here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1838/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>removed cleanup</p>